frontend/src/components/eventForm/eventForm.js

```diff
@@ -77,113 +75,129 @@ const temporalInputField = ({
 
 export const EventForm = props => {
   const { handleChange, classes, values,
-    schools, isSubmitting, touched, errors } = props;
+    schools, isSubmitting, touched, errors, user } = props;
+  console.log(user);
   return ( 
     <Form name='createEventForm'
       className={classes.main_form}
       id='main_form' >
-      <Typography 
-        variant='display1'
-        children="Create An Event"
-      />
-      <Grid container justify='center' >
-        <Grid container item xs='6' direction='column' >
-          <Grid item>
-            <Field 
-              name='eventName'
-              label='Event Name'
-              id='event_name' 
-              variant='outlined'
-              type='text'
-              margin='dense'
-              required
-              onChange={handleChange}
-              component={inputField}
-              helperText={touched.event_name && errors.event_name}
-              />
+        <Grid container direction='row'   style={{height: "60px"}} >
+          <Grid item xs='1' style={{marginTop: "12px"}}>
+            <img alt="icon" src={icon} className={classes.icon}/> 
           </Grid>
+          <Grid item xs='1' style={{marginTop: "15px"}}>
+            <Typography style={{fontFamily: "Raleway", fontWeight: '600'}}
+              variant='display1'>studY</Typography>
+          </Grid>
+          <Grid item xs='6'></Grid>
+          <Grid item xs='4' style={{paddingRight: "20px"}}> 
+            <FormControl fullWidth margin='normal'>
+              <Button
+                type='submit'
+                variant="contained"
+                children='Save Event'
```

Comment:
I think 'create Event' would be describe the action better rather than 'save Event' but it's fine

Reply:
Thanks for the feedback! May be you are right about 'create Event'. I will change it upon my next PL.